src/main/java/com/aws/greengrass/logmanager/CloudWatchLogsUploader.java

```diff
@@ -143,7 +143,7 @@ private boolean uploadLogs(String logGroupName, String logStreamName, List<Input
             return true;
         } catch (ResourceNotFoundException e) {
             // Handle no log group/log stream
-            logger.atError().cause(e).log("Unable to find log group- {} or log stream - {}. Creating them now.",
+            logger.atInfo().cause(e).log("Unable to find log group- {} or log stream - {}. Creating them now.",
```

Comment:
We should probably update this log to say that we will be creating the log group/stream.